src/components/DashboardAdmin/index.scss

```diff
@@ -1,14 +1,84 @@
 .admin {
-  padding: 7%;
+  padding: 5%;
```

Comment:
can you use another unit measurement for padding? i.e. em or rem would be better.